src/app/products/manage-tax-groups/create-tax-group/create-tax-group.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="container">
+<mat-card>
+<form [formGroup]="TaxGroupForm">
+  <mat-card-title>
+    <h3>Create Tax Group</h3> <hr>
+  </mat-card-title>
+  <mat-card-content>
+    <span class="namefield">Name*
+      <mat-form-field appearance="outline">
+        <input matInput required formControlName='name'>
+        <mat-error *ngIf="TaxGroupForm.controls.name.hasError('required')">
+          Name is <strong>required</strong>
+        </mat-error>
+      </mat-form-field>
+    </span>
+
+    <h1>Tax Components</h1> <br>
+    <div fxLayout="row wrap" fxLayoutGap="30%" fxLayout.lt-md="column">
+       <h2>Tax Component</h2>
+        <h2>Start Date</h2>
+        <button mat-button (click)='addTaxcomponent()' class="addbutton">
+        <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;Add
+      </button>
+    </div>
+
+    <div formArrayName="taxComponents"
+        *ngFor="let componentdetail of taxComponents.controls;
+        let componentIndex = index;">
+      <div fxLayout="row wrap" fxLayoutGap="10%" fxLayout.lt-md="column"
+            [formGroupName]="componentIndex">
+        <mat-form-field appearance="outline">
+
+          <mat-select name="Tax Component" formControlName='taxComponentId'
+                      placeholder="Tax Component">
+            <mat-option *ngFor="let taxComponent of taxComponentData"
+                        [value]="taxComponent.id">
+              {{taxComponent.name}}
```

Comment:
Ensure proper indentation.
```suggestion
              {{ taxComponent.name }}
```